### PR TITLE
temporary pr to increase test coverage of ripsaw

### PR DESCRIPTION
**Pull request by aakarshg**
Depends-On: 316

### CODE REVIEW

File: 1.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+hi
```
Comment (dry923) [inline 1.txt]: Hello

Comment (jtaleric) [reply 1.txt]: Hai

**Comment by dry923** [comment]
looks like ci had a bunch of issues pushing to quay. re-running

**Comment by dry923** [comment]
Closing this as this was just for added testing of the ripsaw pr

**Pull request closed without merging**